src/services/user-group/services/courses.js

```diff
@@ -1,10 +1,14 @@
 const { authenticate } = require('@feathersjs/authentication');
+const {
+	iff, isProvider, discard, disallow, keepInArray, keep,
```

Comment:
Codacy found an issue: ['isProvider' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5941548)